dlp/src/test/java/dlp/snippets/JobsTests.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package dlp.snippets;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.concurrent.ExecutionException;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(JUnit4.class)
+public class JobsTests {
+
+    private static final Pattern JOB_ID_PATTERN = Pattern.compile("projects/.*/dlpJobs/i-\\d+");
+    private static final String PROJECT_ID = System.getenv("GOOGLE_CLOUD_PROJECT");
+    private static final String GCS_PATH = "gs://" + PROJECT_ID + "/dlp";
```

Comment:
I think you were probably right the fist time and this should be a provided env var. More generally, any time we ask the user to provide a specific resources we should probably be asking for an environment variable in a test. 

This probably applies to `PUB_SUB_TOPIC_ID` and `PUB_SUB_SUBSCRIPTION_ID` as well. 

If you want to address this in this PR, I can walk you through how to set these during our next 1:1. If not, please open an issue and assign it to yourself and we'll fix at a later time.

Reply:
Decided to keep the env vars. Let's go over how to set these for kokoro on Monday.